# Action that request a trajectory for reaching a given grasping pose

# Goal
geometry_msgs/TransformStamped grasping_pose
geometry_msgs/TransformStamped pre_grasping
geometry_msgs/TransformStamped grasping_pose2
geometry_msgs/TransformStamped pre_grasping2
int32 number_goals
string arm      # Do I need to specify the arm here??
---
# Result
sensor_msgs/JointState[] trajectory 
---
# Feedback
string sim_status
sensor_msgs/JointState[] sim_trajectory
